src/components/chatX/ChatXInput.js

```diff
@@ -2,14 +2,24 @@ import React from 'react';
 import TextInput from '../common/TextInput';
 
 const ChatXInput = ({ content, onChange, onSave }) => {
+
+    const getValue = (event) => {
+        onChange(event.target.value);
+    };
+
+    const onClick = (event) => {
+        event.preventDefault();
+        onSave();
+    };
+
```

Comment:
dom-events are handled in ChatXInput component

Reply:
👍